src/main/webapp/assets/content/json/irc.json

```diff
@@ -351,6 +351,78 @@
       "linkedin": "charininanayakkara"
     }
   },
+  {
+    "name": "Sanjanee Waniganeththi",
+    "description": "Nuclear Physics",
+    "image": "sanajee.jpg",
+    "social": {
+      "linkedin": "sanjanee-waniganeththi-92b3a3111"
+    }
+  },
+  {
+    "name": "Scynthya Nirukshan",
+    "description": "Bioscience Engineering",
+    "image": "scynthya.jpg",
+    "social": {
+      "linkedin": "gertrude-scynthya-nirukshan-a213476a"
+    }
+  },
+  {
+    "name": "Sumudu Herath",
+    "description": "Civil Engineering",
+    "image": "sumudu.jpg",
+    "social": {
+      "linkedin": "sumuduherath"
+    }
+  },
+  {
+    "name": "Maneesha Perera",
+    "description": "Engineering",
+    "image": "maneesha.jpg",
```

Comment:
One of the previous images is overwritten by this image. Could you please rename this image to maneeshaperera.jpg and restore the deleted image(https://sefglobal.org/assets/img/profiles/maneesha.jpg)

Reply:
@Piumal1999 Sorry, that was my own initiative - I thought the pictures I downloaded were more in line with the formats that are used for most of the pictures. I have restored the previous versions and renamed the new pictures that I added, as per your request.

Reply:
Oh sorry I couldn't mention about it earlier in the issue. The problem was because we had few other people with the same name. Anyway now the image files are correct. But I think you have forgotten to update it on the json file (line 381 and line 389 ). Update it and finally we can merge the PR.

Reply:
```suggestion
    "image": "maneeshaperera.jpg",
```

Reply:
I think I misunderstood: I thought you wanted the previous pictures used, so I kept the names in the json. So you want me to change the json to include the two new images? To be sure: I mean using maneeshaperera.jpg and madhushibandara.jpg in the json file instead of maneesha.jpg and madhushi.jpg?

Reply:
OK, I see your change suggestion now, I will update the two image names in the json file.

Reply:
> So you want me to change the json to include the two new images? To be sure: I mean using maneeshaperera.jpg and madhushibandara.jpg in the json file instead of maneesha.jpg and madhushi.jpg

Yeah.